x-pack/plugins/security_solution/public/management/pages/response_actions/view/response_actions_list.tsx

```diff
@@ -0,0 +1,422 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import {
+  CriteriaWithPagination,
+  EuiAvatar,
+  EuiBadge,
+  EuiBasicTable,
+  EuiButtonIcon,
+  EuiDescriptionList,
+  EuiEmptyPrompt,
+  EuiFacetButton,
+  EuiFlexGroup,
+  EuiFlexItem,
+  EuiHorizontalRule,
+  EuiScreenReaderOnly,
+  EuiText,
+  EuiToolTip,
+  RIGHT_ALIGNMENT,
+} from '@elastic/eui';
+
+import type { HorizontalAlignment } from '@elastic/eui';
+import React, { memo, useCallback, useMemo, useState } from 'react';
+import { FormattedMessage } from '@kbn/i18n-react';
+import styled from 'styled-components';
+import { FormattedDate } from '../../../../common/components/formatted_date';
+import {
+  ActionDetails,
+  LogsEndpointActionResponse,
+  EndpointActivityLogAction,
+} from '../../../../../common/endpoint/types';
+import type { EndpointActionListRequestQuery } from '../../../../../common/endpoint/schema/actions';
+import { AdministrationListPage } from '../../../components/administration_list_page';
+import { ManagementEmptyStateWrapper } from '../../../components/management_empty_state_wrapper';
+import { useGetEndpointActionList } from '../../../hooks';
+import { OUTPUT_MESSAGES, TABLE_COLUMN_NAMES, UX_MESSAGES } from '../translations';
+import { MANAGEMENT_PAGE_SIZE_OPTIONS } from '../../../common/constants';
+import { useTestIdGenerator } from '../../../hooks/use_test_id_generator';
+
+// Truncated usernames
+const StyledFacetButton = styled(EuiFacetButton)`
```

Comment:
I don't think this is the correct Component to use for showing the User avatar...  

I had created something for use in the Artifact Card, which Candace also copied and is using in Policy List:

https://github.com/elastic/kibana/blob/e92ed9ff32c7317790882c0f8ca3ce34b8373cc4/x-pack/plugins/security_solution/public/management/pages/policy/view/policy_list.tsx#L186-L197

I would propose lifting that code into a reusable component (I thought that had already been done - sorry) and then using it here in this list component as well.

Reply:
Yeah, so I did use the one you suggest. Same as in the policy table. The problem is things within `FlexItem`s don't truncate, even if we use EuiText with truncate class 😢. So I went with this component as this can be just placed in the cell without `FlexItem`s.

Reply:
Thats strange. I have been able to truncate items in flex items using the `eui-textTruncate` class name. Did you use a `div` to wrap the content?

Reply:
So I tried this:

```
<div>
              <EuiFlexGroup responsive={false} gutterSize={'xs'} alignItems="center">
                <EuiFlexItem grow={false}>
                  <EuiAvatar name={userId} data-test-subj={'created-by-avatar'} size="s" />
                </EuiFlexItem>
                <EuiFlexItem grow={false}>
                  <EuiText
                    size="s"
                    className="eui-textTruncate eui-fullWidth"
                    data-test-subj="created-by-name"
                  >
                    {'userIduserIduserId'}
                  </EuiText>
                </EuiFlexItem>
              </EuiFlexGroup>
            </div>
```

and this:

```
<EuiFlexGroup responsive={false} gutterSize={'xs'} alignItems="center">
              <EuiFlexItem grow={false}>
                <EuiAvatar name={userId} data-test-subj={'created-by-avatar'} size="s" />
              </EuiFlexItem>
              <EuiFlexItem grow={false}>
                <div>
                  <EuiText
                    size="s"
                    className="eui-textTruncate eui-fullWidth"
                    data-test-subj="created-by-name"
                  >
                    {'userIduserIduserId'}
                  </EuiText>
                </div>
              </EuiFlexItem>
            </EuiFlexGroup>
```

Neither worked. Maybe I'm doing it wrong? I'll keep the facet button for now.
The policy table also has the usernames non-truncated by the way.

We can also update this in a follow-up PR.